python/pfs/drp/stella/readLineList.py

```diff
@@ -10,7 +10,8 @@
 class ReadLineListConfig(Config):
     """Configuration for ReadLineListTask"""
     lineListFiles = ListField(dtype=str, doc="list of names of linelist files",
-                              default=["Ar.txt", "Hg.txt", "Kr.txt", "Ne.txt", "Xe.txt", "skyLines.txt"])
+                              default=["HgCd.txt", "Ar.txt", "Hg.txt", "Kr.txt",
+                                       "Ne.txt", "Xe.txt", "skyLines.txt"])
```

Comment:
Including `HgCd.txt` along with `Hg.txt` means that some lines will be duplicated.